projects/wave-core-new/src/lib/datasets/draw-features/draw-features.component.ts

```diff
@@ -41,7 +41,7 @@ export class DrawFeaturesComponent implements OnDestroy, OnInit {
     // the current feature type
     selectedFeatureType: ResultType = ResultTypes.POINTS;
     // the corresponding open layers geometry type
-    olGeometryType: OlGeometryType = OlGeometryType.POINT;
+    olGeometryType: string = OlGeometryType.POINT;
```

Comment:
It was always a string, but just now the compiler said that we could use OlGeometryType as a type – because it is not a type. It is just a thing that has some constants.